### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by STDio96**
Redux structure:
- store;
- actions;
- reducers;
- dummy data

### CODE REVIEW

File: src/redux/actions/data.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { response } from '../response' // TODO: remove on production
+import { SET_LOADED, SET_DATA } from '../actionConstants'
+
+// TODO: mb create action for each subarray
```
Comment (dkupchyk) [inline src/redux/actions/data.js:4]: Not sure that that comment is necessary

File: src/redux/reducers/filters.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { TOGGLE_PROVIDER, SET_CATEGORY } from '../actionConstants'
+
+const initialState = {
+    selectedCategory: null, // only 1 category
+    selectedProviders: [], // multiple providers
+}
+
+const filters = (state = initialState, action) => {
+    /*
```
Comment (dkupchyk) [inline src/redux/reducers/filters.js]: Delete unnecessary code.

File: src/redux/actionConstants.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export const SET_DATA = 'data/set_data'
```
Comment (dkupchyk) [inline src/redux/actionConstants.js]: Would move this file to actions folder

File: src/redux/reducers/index.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { combineReducers } from 'redux'
```
Comment (dkupchyk) [inline src/redux/reducers/index.js:1]: Not sure whether there is a unnecessity to create a file for this, I would personaly just initialize rootReducer  in the _store.js_ file and use it there, but if you think that this structure is better than okay :)

File: src/redux/actions/data.js (hunk 1 above)
Comment (STDio96) [reply src/redux/actions/data.js:4]: I left it just not to forget about such an opportunity :)

File: src/redux/reducers/index.js (hunk 4 above)
Comment (STDio96) [reply src/redux/reducers/index.js:1]: Here's what I think: we have a separate folder for reducers and `rootReducer` is a reducer as well, so I moved it to the /reducers folder :)

File: src/redux/actions/data.js (hunk 1 above)
Comment (STDio96) [reply src/redux/actions/data.js:4]: removed ✅

File: src/redux/reducers/filters.js (hunk 2 above)
Comment (STDio96) [reply src/redux/reducers/filters.js]: removed ✅

File: src/redux/actionConstants.js (hunk 3 above)
Comment (STDio96) [reply src/redux/actionConstants.js]: not sure if it's necessary, but - done ✅

File: src/redux/reducers/index.js (hunk 4 above)
Comment (dkupchyk) [reply src/redux/reducers/index.js:1]: good, lets leave it there

**Review by dkupchyk — APPROVED** [review]

**Pull request merged**